packages/invoices/src/InboundShipment/DetailView/reducer/reducer.test.ts

```diff
@@ -0,0 +1,3 @@
+describe('Inbound shipment DetailView Reducer', () => {
+  it('is a placeholder', () => {});
+});
```

Comment:
I'm keen to wait and see if changes are needed to the reducer. currently it's a copy and paste which already has tests